static/locales/en-US.yaml

```diff
@@ -131,26 +130,26 @@ Settings:
     Current Invidious Instance: Current Invidious Instance
     # $ is replaced with the default Invidious instance
     The currently set default instance is $: The currently set default instance is $
-    No default instance has been set: No default instance has been set
-    Current instance will be randomized on startup: Current instance will be randomized on startup
+    No default instance has been set: No default instance  set
```

Comment:
This line has a double space.

Reply:
Asides from these, I'm ready to approve these changes.

Reply:
Good catch :)